packages/pages/src/common/src/parsers/sourceFileParser.ts

```diff
@@ -207,4 +208,115 @@ export default class SourceFileParser {
   getAllText(): string {
     return this.sourceFile.getFullText();
   }
+
+  /**
+   * For example, we can do getVariablePropertyByType("ModuleConfig", "name")
```

Comment:
I think this will be brittle to use. There's nothing forcing a user to actual type their config. They should, but they don't have to. So if we're solely relying on the type to find the config then this will fail. The config name is specific though, so we can just look for that.

Reply:
Oh I see. By "The config name is specific" do you mean, they'll always name the const `config` and if so why / how can we guarantee this?

Reply:
Yes. We can guarantee it because it's convention for how to use a module. I mean it's technically not required because name is the only property which is optional, so you'd have to look for an export named config, otherwise it doesn't exist and we end up using the filename.